### DISCUSSION TITLE
Downtime for https://search.inetol.net

### DISCUSSION THREAD

**Discussion by inetol** [searxng/searx-instances#580] [Report instance downtime]
A few hours ago we were informed that the number of requests to the SearXNG instance was slightly exceeding the average, there was a scheduled maintenance to move the instance to another region this same day (oddly enough) with which there were no incidents. However, shortly thereafter the number of requests increased significantly, so much so that the uWSGI server was running out of capacity and requests were starting to pile up on the main server.

To avoid interference with other services, the instance has been temporarily limited until requests return to normal levels.

`Aug. 22, 2024, 17:45 GMT`: The web server has been updated to accommodate the experimental filtering subsystem, it is possible that some legitimate users are being filtered out. In a couple of days it is expected to be polished enough so that it will not give false positives.

We consider that the outage has been resolved at this point.

**Comment by return42** [comment]
> To avoid interference with other services, the instance has been temporarily limited until requests return to normal levels.

Yes, there was a bot flood / we discussed in Matrix room "SearXNG public instances owner".

For all coming around and having similar issues: please check that your bot detection (also known as limiter) is set up correctly: https://github.com/searxng/searxng/issues/3191#issuecomment-2202729527

**Reply by inetol** [reply]
Please invite me to that room because I'm not in: `@inetol:mozilla.org`

Thanks.

**Reply by return42** [reply]
[didn't we invite you?](https://github.com/searxng/searx-instances/issues/424#issuecomment-1775122966) / sorry .. but now you should received an invitation.
